bin/include/jvmdefaults.sh

```diff
@@ -98,7 +77,7 @@ getJavaSpecificOpts() {
           --add-opens=java.base/sun.security.ssl=ALL-UNNAMED \
           --add-opens=java.base/sun.security.x509=ALL-UNNAMED \
           --add-opens=java.sql/java.sql=ALL-UNNAMED \
-          ${current_value}"
+          $current_value"
```

Comment:
Why did we remove the brackets?

Reply:
An accidental error made during code formatting. Will revert this change. Thank you for pointing it out